### PR TITLE
add django-rq

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by helloreuben** [comment]
@sannykr  needs django configurations like https://github.com/conda-forge/django-viewflow-feedstock/blob/master/recipe/meta.yaml
- python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['django_rq']); django.setup(); import django_rq;

**Comment by sannykr** [comment]
@helloreuben : Thanks. I will add the run_test.py file to the recipe.

**Comment by cshaley** [comment]
@sannykr Here's the bare minimum working import test.

```
import django
from django.conf import settings
settings.configure(INSTALLED_APPS=['django_rq'], RQ_QUEUES={})
django.setup()
import django_rq
```

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : could you please review and merge if everything looks okay.

File: recipes/django-rq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['django_rq'], RQ_QUEUES={}); django.setup(); import django_rq"    
+
+about:
+  home: https://github.com/ui/django-rq
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```
Comment (dougalsutherland) [inline recipes/django-rq/meta.yaml]: `LICENSE.txt` is already included in the source file, so please remove it from the recipe and change to just `LICENSE.txt`. :)

**Comment by dougalsutherland** [comment]
LGTM after the license note above.

File: recipes/django-rq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (dougalsutherland) [inline recipes/django-rq/meta.yaml]: This should be `noarch: python` too.

**Comment by dougalsutherland** [comment]
LGTM after comments above.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge. There is some issue with python 3.5 on windows, that's why only that check failed.

**Pull request merged**